token/cli/src/clap_app.rs

```diff
@@ -962,6 +963,52 @@ pub fn app<'a, 'b>(
                 .arg(transfer_lamports_arg())
                 .offline_args_config(&SignOnlyNeedsTransferLamports{}),
         )
+        .subcommand(
+            SubCommand::with_name(CommandName::InitializeGroup.into())
+                .about("Initialize group extension on a token mint")
+                .arg(
+                    Arg::with_name("token")
+                        .validator(is_valid_pubkey)
+                        .value_name("TOKEN_MINT_ADDRESS")
+                        .takes_value(true)
+                        .required(true)
+                        .index(1)
+                        .help("The token address of the group account."),
+                )
+                .arg(
+                        Arg::with_name("max_size")
+                        .validator(is_amount)
+                        .value_name("MAX_SIZE")
+                        .takes_value(true)
+                        .required(true)
+                        .index(2)
+                        .help("The number of members in the group."),
+                    )
```

Comment:
Just an idea here, but what if we allowed for the keyword `max` here, and we'd default that to `U32::MAX`?

Reply:
I think to do this we would need to create a new validator in the solana-clap-utils like `is_amount_or_max`. I am also not sure how common of a use case that would be. I think its good as is 👍 

After this, I will work my way through the rest of the commands as separate PRs. 😎